.travis.yml

```diff
@@ -54,44 +75,14 @@ cache:
 
 matrix:
   include:
-    # lint, docs and coverage first as they're more likely to find issues
-    - python: "2.7"
-      env: TWISTED=latest SQLALCHEMY=latest TESTS=lint
-    - python: "2.7"
-      env: TWISTED=latest SQLALCHEMY=latest TESTS=docs
-    - python: "2.7"
-      env: TWISTED=latest SQLALCHEMY=latest TESTS=coverage
-
-    # Test different versions of SQLAlchemy
-    - python: "2.7"
-      env: TWISTED=15.5.0 SQLALCHEMY=0.6.0 TESTS=trial
-    - python: "2.7"
-      env: TWISTED=15.5.0 SQLALCHEMY=0.6.8 TESTS=trial
-    - python: "2.7"
-      env: TWISTED=15.5.0 SQLALCHEMY=0.7.0 TESTS=trial
-    - python: "2.7"
-      env: TWISTED=15.5.0 SQLALCHEMY=0.7.4 TESTS=trial
-    - python: "2.7"
-      env: TWISTED=15.5.0 SQLALCHEMY=0.7.8 TESTS=trial
-    - python: "2.7"
-      env: TWISTED=15.5.0 SQLALCHEMY=latest TESTS=trial
-
-    # add extra tests in separate jobs
-    - python: "2.7"
-      env: TWISTED=latest SQLALCHEMY=latest TESTS=js
-
+    - python: "2.6"
+      env: TWISTED=14.0.2 TESTS=trial_worker
+    - python: "2.6"
+      env: TWISTED=15.4.0 TESTS=trial_worker
```

Comment:
This non-obvious to me from `trial_worker` name, but explains why:

1. `TWISTED=latest` is not specified for it (latest Twisted doesn't support Python 2.6).
2. `master` is not installed in this configuration (Python-level master-worker communication, i.e. local worker, works only with buildbot-worker on Python 2.7, which is tested in `TESTS=trial` configuration).

Maybe we should add comment regarding this?